src/components/application_manager/src/state_controller_impl.cc

```diff
@@ -88,7 +88,10 @@ void StateControllerImpl::SetRegularState(ApplicationSharedPtr app,
     return;
   }
 
-  if (app->is_resuming() && !IsResumptionAllowed(app, state)) {
+  const bool app_is_resuming = app->is_resuming();
+  const bool is_resumption_allowed = IsResumptionAllowed(app, state);
+
+  if (app_is_resuming && !is_resumption_allowed) {
```

Comment:
@AByzhynar do we need some log here?

Reply:
@ydementieiev I think no as all detailed logs present in IsResumptionAllowed(..)